Realm/RLMArray.h

```diff
@@ -492,6 +492,18 @@ __attribute__((warn_unused_result));
  */
 - (instancetype)freeze;
 
+/**
+ Returns a live, mutable version of this frozen collection.
+
+ This method resolves a reference to a live copy of the same frozen collection.
+ If a called on a non-frozen collection, will return itself.
+
+ @warning Holding onto a thawed collection for an extended period while performing write
```

Comment:
Incorrect copy/paste.

Reply:
Removed in later commit.
What makes it incorrect, though? Is it because the thread safe references is no longer a liability once resolved?

Reply:
A thawed collection is just a normal live collection and this warning is about frozen collections.

Reply:
Got it, thanks. I misunderstood the nature of what made frozen collections dangerous to hold onto, then.